scripts/scripts.js

```diff
@@ -322,6 +334,7 @@ async function loadEager(doc) {
  * @param {Element} doc The container element
  */
 async function loadLazy(doc) {
+  autolinkModals(doc);
```

Comment:
move this method call towards end of loadLazy()

Reply:
This is addressed.